Visualización de reporte luego de completar orden en el PDV

**Prueba en el contenedor:** 
http://20.12.0.2:8080/#
http://20.12.0.2:55810/webui/timeout.zul

Se requiere ver el reporte de factura al seleccionar la opción de **_Vista Previa_** en el PDV, debido que se quiere imprimir el mismo. Pero al seleccionar la opción, genera un mensaje que no permite la visualización de dicho reporte.

**Adjunto imagen**
![Screenshot 2022-11-11 9 24 31 AM](https://user-images.githubusercontent.com/44775211/201349939-514a5cae-e426-4ce1-9d5a-d877fad99f06.png)

Cuando se busca la factura en ADempiere, y se selecciona la opción de imprimir, se visualiza el reporte. Adjunto imagen
![Screenshot 2022-11-11 9 29 09 AM](https://user-images.githubusercontent.com/44775211/201350121-11e961c5-9d67-418e-9e3f-dd2c0f3973ec.png)

**Documento por Cobrar:** NE-362

**Usuario:** Cajero
**Clave:** 123456

Inicialmente faltaba permisos al rol, ahora genera el siguente error en el backend
```
===========> ReportStarter.startProcess: ReportStarter.startProcess: Can not run report - Error executing SQL statement for: Factura CxC. [66]
===========> ReportStarter.startProcess: ReportStarter.startProcess: Can not run report - Error executing SQL statement for: Factura CxC. [66]
[WARNING] [c3p0] A PooledConnection that has already signalled a Connection error is still in use!
[WARNING] [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners!org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2497)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2233)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:446)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:370)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:149)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:100)
	at com.sun.proxy.$Proxy4.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdateEx(DB.java:1119)
	at org.compiere.util.DB.executeUpdateEx(DB.java:1092)
	at org.compiere.report.ReportStarter.reportResult(ReportStarter.java:961)
	at org.compiere.report.ReportStarter.startProcess(ReportStarter.java:689)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:162)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:107)
	at org.compiere.process.ServerProcessCtl.startProcess(ServerProcessCtl.java:358)
	at org.compiere.process.ServerProcessCtl.run(ServerProcessCtl.java:227)
	at org.compiere.process.ServerProcessCtl.process(ServerProcessCtl.java:114)
	at org.compiere.print.ServerReportCtl.runJasperProcess(ServerReportCtl.java:130)
	at org.compiere.print.ServerReportCtl.startDocumentPrint(ServerReportCtl.java:74)
	at org.compiere.print.ServerReportCtl.start(ServerReportCtl.java:193)
	at org.compiere.process.ServerProcessCtl.run(ServerProcessCtl.java:233)
	at org.eevolution.service.dsl.ProcessBuilder.run(ProcessBuilder.java:221)
	at org.eevolution.service.dsl.ProcessBuilder.lambda$execute$0(ProcessBuilder.java:268)
	at org.compiere.util.Trx.run(Trx.java:529)
	at org.compiere.util.Trx.run(Trx.java:497)
	at org.eevolution.service.dsl.ProcessBuilder.execute(ProcessBuilder.java:266)
	at org.spin.grpc.service.BusinessDataServiceImplementation.runProcess(BusinessDataServiceImplementation.java:334)
	at org.spin.grpc.service.PointOfSalesServiceImplementation.printPreview(PointOfSalesServiceImplementation.java:1056)
	at org.spin.backend.grpc.pos.StoreGrpc$MethodHandlers.invoke(StoreGrpc.java:5308)
	at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:171)
	at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:283)
	at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:710)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.postgresql.util.PSQLException: ERROR: column reference "discount" is ambiguous
```

--- a/components/ADempiere/Form/VPOS/Options/index.vue
+++ b/components/ADempiere/Form/VPOS/Options/index.vue
@@ -1352,11 +1352,14 @@ export default {
       }
     },
     printPreview() {
-      const orderUuid = this.currentOrder.uuid
       const posUuid = this.currentPointOfSales.uuid
+      const orderUuid = this.currentOrder.uuid
 
       this.isLoadingPrintPreview = true
-      this.$store.dispatch('printTicketPreviwer', { posUuid, orderUuid })
+      this.$store.dispatch('printTicketPreviwer', {
+        posUuid,
+        orderUuid
+      })
         .then(response => {
           const { processLog } = response
           if (!this.isEmptyValue(processLog)) {

